puppet/manifests/development.pp

```diff
@@ -5,8 +5,6 @@
 $extensions = sz_extensions('/vagrant/extensions')
 $php_extensions = [ 'curl', 'gd', 'mysql' ]
 
-Class['mysql'] -> Package['php5-mysql']
-
```

Comment:
Nah we don't need this here anymore as I handled it here instead: https://github.com/Chassis/Chassis/pull/257/files#diff-ebb1484b5498abe97a80e94aac9365cdR58